nuts-network/gossip-overflow/run-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+source ../../util.sh
+
+echo "------------------------------------"
+echo "Cleaning up running Docker containers and volumes, and key material..."
+echo "------------------------------------"
+docker-compose down
+docker-compose rm -f -v
+
+echo "------------------------------------"
+echo "Starting Docker containers..."
+echo "------------------------------------"
+docker-compose up -d
+waitForDCService nodeA
+waitForDCService nodeB
+
+# Wait for Nuts Network nodes to build connections
+sleep 1
+
+# create 200 new DID documents on each node
+echo "------------------------------------"
+echo "Creating transactions"
+echo "------------------------------------"
+
+for i in {1..200}
+do
+   curl -s -X POST http://localhost:11323/internal/vdr/v1/did >/dev/null
+done
+
+echo "------------------------------------"
+echo "Performing assertions..."
+echo "------------------------------------"
+sleep 15
```

Comment:
This now becomes a performance test that asserts it completes within 15 seconds (and it always blocks for 15 seconds, even when it's faster). We should probably check every 1 second or so in a loop and time-out after 15 seconds. You can probably copy most of `waitForDCService`.

Reply:
used API to do get diagnostics, better to use docker compose exec?

Reply:
docker compose exec uses the same API, but then using the client. So potato-potato, I think?